Cargo.toml

```diff
@@ -1,16 +1,40 @@
 [package]
 name = "hades252"
-version = "0.1.0"
-authors = ["kev <kevtheappdev@gmail.com>"]
+version = "0.1.1"
+authors = [
+  "kev <kevtheappdev@gmail.com>", "zer0 <matteo@dusk.network>",
+  "Victor Lopes <victor@dusk.network>", "kr0 <c.perezbaro@gmail.com>"
+]
 edition = "2018"
 
 [dependencies]
 hex = "0.3.1"
 sha2 = "0.8"
-curve25519-dalek = { version = "1.0.1", features = ["serde"] }
-merlin = "1.1"
+curve25519-dalek = { version = "1.2.3", features = ["serde", "avx2_backend"] }
+merlin = "1.2.0"
 rand = "0.6"
```

Comment:
Correct me if I'm wrong. But since we use rand only for tests, we could add it as a `dev-dependency` and add a `#[cfg(feature = rand)]` in the tests where is needed.

Reply:
If it's used only in test, we can definitely add as `dev-dependecy`, you're not wrong!

Reply:
Also, `hex` and `sha2` dependencies are not used anywhere. 
I'm removing them.

Reply:
Done.